docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -391,6 +391,21 @@ below.
   This will make the mean slightly larger than it should be, but so will any
   strategy we pick.
 
+Model Randomness
+~~~~~~~~~~~~~~~~
```

Comment:
Is this supposed to be a subsection of "Stratifying Exposure Variables by Baseline Intervention Coverage" (which it currently is), or a separate section (in which case the section level should be +++++++)?

Reply:
I was just thinking whether we fixed this for the BEP costs in our bep model?

Reply:
Good catch! I updated to "+++++"